joint_limits:
    flipper_fl_j:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 1.1
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 255
      min_effort: -255

  flipper_fr_j:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 1.1
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 255
      min_effort: -255

flipper_rl_j:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 1.1
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 255
      min_effort: -255

flipper_rr_j:
      has_position_limits: false
      min_position: 0
      max_position: 0
      has_velocity_limits: true
      max_velocity: 1.1
      has_acceleration_limits: false
      max_acceleration: 0.0
      has_jerk_limits: false
      max_jerk: 0
      has_effort_limits: true
      max_effort: 255
      min_effort: -255
